README.md

```diff
@@ -49,7 +49,7 @@ talk at Next 2018:
 [![Video: Building Go Applications for the Open Cloud (Cloud Next '18)](https://img.youtube.com/vi/_2ZwhvIkgek/0.jpg)][video]
 
 [announcement blog post]: https://blog.golang.org/go-cloud
-[godoc]: https://godoc.org/github.com/google/go-cloud
+[godoc]: https://pkg.go.dev/gocloud.dev
```

Comment:
How about https://pkg.go.dev/mod/gocloud.dev ?

Reply:
Also an option. Not sure what they recommend to use. There is also an official badge if you prefer https://pkg.go.dev/badge/.